tests/intfutil_test.py

```diff
@@ -62,6 +62,31 @@
  Ethernet32      up       up     etp9  Servers7:eth0
 """
 
+show_interface_auto_neg_status_output = """\
+  Interface    Auto-Neg Mode    Speed    Adv Speeds    Type    Adv Types    Oper    Admin
+-----------  ---------------  -------  ------------  ------  -----------  ------  -------
+  Ethernet0              N/A      25G           N/A     N/A          N/A    down       up
+ Ethernet32              N/A      40G           N/A     N/A          N/A      up       up
+Ethernet112              N/A      40G           N/A     N/A          N/A      up       up
+Ethernet116              N/A      40G           N/A     N/A          N/A      up       up
+Ethernet120              N/A      40G           N/A     N/A          N/A      up       up
+Ethernet124              N/A      40G           N/A     N/A          N/A      up       up
+"""
+
+show_interface_auto_neg_status_Ethernet0_output = """\
+  Interface    Auto-Neg Mode    Speed    Adv Speeds    Type    Adv Types    Oper    Admin
+-----------  ---------------  -------  ------------  ------  -----------  ------  -------
+  Ethernet0              N/A      25G           N/A     N/A          N/A    down       up
+"""
+
+show_interface_auto_neg_status_eth9_output = """\
+  Interface    Auto-Neg Mode    Speed    Adv Speeds    Type    Adv Types    Oper    Admin
+-----------  ---------------  -------  ------------  ------  -----------  ------  -------
+ Ethernet32              N/A      40G           N/A     N/A          N/A      up       up
+"""
```

Comment:
Can we add some test cases in which the `Auto-Neg Mode`, `Adv Speeds` and `Adv Types` are not `N/A`